components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/reporting-overview.component.ts

```diff
@@ -118,7 +118,7 @@ export class ReportingOverviewComponent implements OnInit, OnDestroy {
 
   onPlatformChanged(platformItem) {
     if (platformItem && platformItem.name) {
-      this.updateUrlWithFilter(platformItem.name, 'platform');
+      this.updateUrlWithFilter(platformItem.name + '*', 'platform_with_version');
```

Comment:
When the platform bubble is selected filter with the `platform_with_version` type with a wildcard for all versions.